src/core/Checkbox/index.tsx

```diff
@@ -14,8 +14,9 @@ export interface CheckboxProps
  * @see https://v4.mui.com/components/checkboxes/
  */
 const Checkbox = (props: CheckboxProps): JSX.Element => {
+  const { stage } = props;
   let newProps: MUICheckboxProps;
-  switch (props.stage) {
+  switch (stage) {
```

Comment:
this was a linter warning that I fixed while trying to resolve other issues. It's not strictly related, but I figured it doesn't hurt to get rid of a linter warning.